Dockerfile

```diff
@@ -0,0 +1,72 @@
+ARG PYTHON_VERSION=3.12-slim
+
+FROM  --platform=$BUILDPLATFORM python:${PYTHON_VERSION} AS project-base
+
+LABEL maintainer=ai-validatie@minbzk.nl \
+    organization=MinBZK \
+    license=EUPL-1.2 \
+    org.opencontainers.image.description="Instrument Registry" \
+    org.opencontainers.image.source=https://github.com/MinBZK/instrument-registry \
+    org.opencontainers.image.licenses=EUPL-1.2
+
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    PIP_NO_CACHE_DIR=off \
+    PIP_DISABLE_PIP_VERSION_CHECK=on \
+    PIP_DEFAULT_TIMEOUT=100 \
+    POETRY_VIRTUALENVS_IN_PROJECT=true \
+    POETRY_NO_INTERACTION=1 \
+    POETRY_HOME='/usr/local'
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl libpq-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN  curl --proto "=https" -sSL https://install.python-poetry.org | python3 -
+
+WORKDIR /app/
+COPY ./poetry.lock ./pyproject.toml ./
+COPY ./instruments/ ./instruments/
+COPY ./script/ ./script/
+
+RUN poetry install
+ENV PATH="/app/.venv/bin:$PATH"
+RUN python script/deploy
```

Comment:
i would reconsider removing this and simply embed the logic in the application lifecycle hook. The application would be more understandable if all logic is inside it.